src/s1ap/handlers/s1ap_encoder.c

```diff
@@ -113,31 +133,37 @@ int s1ap_mme_encode_service_rej(
 
     log_msg(LOG_DEBUG, "Encode Serivce Rej");
     pdu.present = S1AP_PDU_PR_initiatingMessage;
-    pdu.choice.initiatingMessage = calloc(sizeof(InitiatingMessage_t), sizeof(uint8_t));
+    pdu.choice.initiatingMessage = //calloc(sizeof(InitiatingMessage_t), sizeof(uint8_t));
```

Comment:
Does this compile ? I see code comment after =

Reply:
fixed. not sure how it was compiling.